files/mods.json

```diff
@@ -164,6 +164,29 @@
         "packages": ["potioneffects", "redaction", "polyfrostkeystrokes"],
         "hash": "8e3980e5abaf1c5c5adb4dae6f3ed9f4"
     },
+    {
+        "id": "quickjoin",
+        "display": "Quickjoin",
+        "creator": "QWERTZ_EXE",
+        "description": "Quickly join any game on Hypixel!",
```

Comment:
Please don't use exaggerating punctuation, and make this more descriptive - how does this mod accomplish the goal? It adds a menu, right? So say that.